src/commands/apply.js

```diff
@@ -0,0 +1,101 @@
+const log = require('@dhis2/cli-helpers-engine').reporter
+const fs = require('fs-extra')
+const path = require('path')
+
+const { availableCodemods: availableCodemodsLocal } = require('../utils/codemods/availableCodemods')
+const { exec } = require('./apply/exec')
+const { findAvailableCodemodsInNodeModules } = require('../utils/codemods/findAvailableCodemodsInNodeModules')
+const { getCodemodByPackageAndName } = require('../utils/codemods/getPathByPackageAndName')
+const { mergeCodemods } = require('../utils/codemods/mergeCodemods')
+
+module.exports.command = 'apply <files..>'
+module.exports.alias = 'a'
+module.exports.desc = 'Applys a codemod on a given file/directory'
+
+module.exports.builder = yargs =>
+    yargs
+        .positional('files', {
+            describe: 'Files or path the codemod should be applied to',
+            type: 'array',
+        })
```

Comment:
I propose this pattern, and that `<codemod>` is a delimited `package:name` string:

```
module.exports.command = 'apply <codemod> [files..]'
```

And the desc:

```
        .positional('codemod', {
            describe: 'The codemod to apply identified by the format package:codemod, or by a path to a specific codemod file',
            type: 'string',
        })
        .positional('files', {
            describe: 'Paths the codemod should be applied to',
            type: 'array',
            default: '**/*'
        })
```

Also note that I think that the default for `files` should be `**/*`.